pkg/database/migrations/012-insert_as_new.sql

```diff
@@ -0,0 +1,43 @@
+---- This file is Free Software under the Apache-2.0 License
+-- without warranty, see README.md and LICENSES/Apache-2.0.txt for details.
+--
+-- SPDX-License-Identifier: Apache-2.0
+--
+-- SPDX-FileCopyrightText: 2024 German Federal Office for Information Security (BSI) <https://www.bsi.bund.de>
+-- Software-Engineering: 2024 Intevation GmbH <https://intevation.de>
+
+-- create_advisory checks if the new document is newer than the old one.
+CREATE OR REPLACE FUNCTION create_advisory() RETURNS trigger AS $$
+    DECLARE
+        old_id           int;
+        old_rev_length   int;
+        old_release_date timestamptz;
+    BEGIN
+        -- Ensure having an advisories record.
+        INSERT INTO advisories (tracking_id, publisher)
```

Comment:
Instead of the `UPDATE` below we might use this `INSERT` with an `ON CONFLICT` clause that updates the state field to `'new'`